tests/test_multi_gateway.py

```diff
@@ -59,12 +58,16 @@ def test_multi_gateway_coordination(config, image, conn):
     periodic polling.
     """
     stubA, stubB = conn
-
     bdev = "Ceph0"
     nqn = "nqn.2016-06.io.spdk:cnode1"
     serial = "SPDK00000000000001"
     nsid = 10
+    num_subsystems = 2
+
     pool = config.get("ceph", "pool")
+    enable_discovery_controller = config.getboolean("gateway", "enable_discovery_controller")
```

Comment:
```suggestion
    enable_discovery_controller = config.getboolean_with_default("gateway", "enable_discovery_controller", False)
```

If `enable_discovery_controller ` is not defined in conf file, default to False.